packages/sanity/src/core/releases/tool/detail/useBundleDocuments.ts

```diff
@@ -110,10 +111,16 @@ const getActiveReleaseDocumentsObservable = ({
               )
             }),
           )
-        const validation$ = validateDocumentWithReferences(ctx, document$).pipe(
-          map((validationStatus) => ({
+        const validation$ = combineLatest([
+          document$,
+          validateDocumentWithReferences(ctx, document$),
+        ]).pipe(
+          map(([document, validationStatus]) => ({
             ...validationStatus,
-            hasError: validationStatus.validation.some((marker) => isValidationErrorMarker(marker)),
+            document,
+            hasError:
+              !isGoingToUnpublish(document) &&
+              validationStatus.validation.some((marker) => isValidationErrorMarker(marker)),
           })),
```

Comment:
Wdyt if we do something like this, so we avoid going through the `validateDocumentWithReferences` function for documents which are marked to unpublish?

```suggestion
        const validation$ = document$.pipe(
          switchMap((document) => {
            if (isGoingToUnpublish(document)) {
              return of({isValidating: false, validation: []} satisfies ValidationStatus)
            }
            return validateDocumentWithReferences(ctx, of(document)).pipe(
              map((validationStatus) => ({
                ...validationStatus,
              })),
            )
          }),
        )

```

Reply:
Had to change it slightly but this works! fb013cb447599ed8112c1c07f3ef5132665492b4